torch/_dynamo/variables/lists.py

```diff
@@ -44,6 +44,31 @@ def _listlike_contains_helper(items, search, tx, options):
     return result
 
 
+def _index_helper(items, search, tx, args, options):
```

Comment:
Lots of as_python_constant() and ConstantVariable() is sus, I don't believe this will work properly for a lot of types.